src/ros_license_toolkit/package.py

```diff
@@ -43,14 +44,10 @@
 
 def get_spdx_license_name(scan_results: Dict[str, Any]) -> Optional[str]:
     """Get the SPDX license name from scan results."""
-    return next(
-        (
-            _license["spdx_license_key"]
-            for _license in scan_results["licenses"]
-            if _license["score"] >= 99
-        ),
-        None,
-    )
+    if scan_results['percentage_of_license_text'] >=\
+            REQUIRED_PERCENTAGE_OF_LICENSE_TEXT:
```

Comment:
This is literally what `is_license_text_file` does. I think I created it because it is more descriptive and easier to read than this.